lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -320,6 +321,7 @@ def create_post_for_auth_or_purchase(money, payment, options)
 
         add_metadata(post, options)
         add_application_fee(post, options)
+        add_exchange_rate(post, options)
```

Comment:
Wrestled with the idea of putting it here, or just where its called for a purchase.